src/grafana/prometheus/docker/docker-compose.yml

```diff
@@ -8,21 +8,19 @@ services:
   # It could instead, use volume-mounts.
   # Use `docker exec -it prometheus /bin/sh` to logging in the container
   prometheus:
-    build: './prometheus'
+    image: prom/prometheus:v2.7.1
     container_name: 'prometheus'
     volumes:
       - 'scdf-targets:/etc/prometheus/'
```

Comment:
see below about question on service discovery random port issue

Reply:
in regards to the `scdf-targets` -- IIRC that was where the service discovery tool writes out app info and i thought that wasn't going to work in the compose setup. can talk about offline if im misunderstanding